source/accessories/uinverter/hardware-design.md

```diff
@@ -185,6 +185,21 @@ V_\text{amp} &= (\text{amplifier gain}) \times (\text{current} \times R_\text{se
 When the DC supply voltage exceeds 12V, the phase current increases significantly and may push the current sense amplifier in saturation.
 To aviod any damage to components on the AMDC board, the DC supply voltage must be limited to no more than 12V.  
 
+### DC link voltage sensing filter
+
+A resistive potential divider ($R_1$ and $R_2$) is provided for DC bus voltage sensing. 
+To filter out high frequency noise in DC link voltage measurement, an additional capacitor can be placed at the voltage sensing point to form a low pass filter (see image). This filter should not affect the nominal sampling operation of the DC bus voltage, i.e. we should still be able to capture actual variations in DC link voltage.
```

Comment:
This phrasing is describing the capacitor as a hypothetical ("can be placed," "should not affect," "we should still"). However, this capacitor has been implemented.  Please revise the description to be more direct and succinct (the description is also unnecessarily long/rambling).